java/java.editor/src/org/netbeans/modules/java/editor/javadoc/JavadocCompletionItem.java

```diff
@@ -156,10 +157,10 @@ public String toString() {
                 txt, substitutionOffset);
     }
 
-    public static List<CompletionItem> addBlockTagItems(ElementKind kind,
+    public static List<CompletionItem> addBlockTagItems(SourceVersion sourceVersion,ElementKind kind,
             String prefix, int startOffset) {
         
-        List<TagEntry> tags = TagRegistery.getDefault().getTags(kind, false);
+        List<TagEntry> tags = TagRegistery.getDefault().getTags(kind, false,sourceVersion);
```

Comment:
nit: please use a space behind the "," to be consistent with the rest of the file